docs/src/content/docs/zh-cn/reference/configuration.mdx

```diff
@@ -465,7 +465,7 @@ interface HeadConfig {
 
 定义页脚是否应包含上一页和下一页的链接。
 
-页面可以使用 [`prev`](/zh-cn/reference/frontmatter/#prev) 和 [`next`](/zh-cn/reference/frontmatter/#next) frontmatter 字段覆盖此设置或链接文本和/或 URL。
+页面可以使用 [`prev`](/zh-cn/reference/frontmatter/#prev) 和 [`next`](/zh-cn/reference/frontmatter/#next) frontmatter 字段覆盖此设置或链接文本 **和/或** URL。
```

Comment:
Is this change related to a specific commit? I don't see bolded text in the English version so I was wondering what was the reasoning for the chamge.

Reply:
Thanks. I restored it.